src/open_inwoner/accounts/views/profile.py

```diff
@@ -247,28 +247,44 @@ def form_valid(self, form):
             return HttpResponseRedirect(self.get_success_url())
 
         # write changes to API's; abort saving if write fails
-        # we treat `api_update_ok` as True if the relevant service is not configured at all
-        esuite_update_ok, openklant_update_ok = True, True
         klanten_config = KlantenSysteemConfig.get_solo()
+        failed_services = []
         if klanten_config.has_api_service_configured(KlantenServiceType.ESUITE):
-            esuite_update_ok = self.update_klant_via_esuite(
-                {k: form.cleaned_data[k] for k in form.changed_data}, user
-            )
+            try:
+                self.update_klant_via_esuite(
+                    {k: form.cleaned_data[k] for k in form.changed_data}, user
+                )
+            except Exception:
+                logger.exception("eSuite failed during profile update for %(user)s" % user)
```

Comment:
I've found it's good practice to not interpolate parameters in the exception string, it makes it harder to dedupe at Sentry level. I would put the user in the `extra` dictionary:


```suggestion
                logger.exception("eSuite failed during profile update", extra={"user":user})
```